docs/contributions.md

```diff
@@ -1,157 +1,200 @@
 # Contribution Guidelines
 
-
 First off, thanks for taking the time to contribute! 🎉
 
 The following document is a rule set of guidelines for contributing.
 
 ## Table of Contents <!-- omit from toc -->
 
 - [Contribution Guidelines](#contribution-guidelines)
+  - [What and when to contribute](#what-and-when-to-contribute)
+  - [Reporting bugs](#reporting-bugs)
   - [Code Contributions](#code-contributions)
-      - [Step 1: Fork](#step-1-fork)
-      - [Step 2: Branch](#step-2-branch)
-      - [Step 3: Commit](#step-3-commit)
-      - [Step 4: Sync](#step-4-sync)
-      - [Step 5: Push](#step-5-push)
-      - [Step 6: Pull Request](#step-6-pull-request)
-  - [Commit message guidelines](#commit-message-guidelines)
+    - [Step 1: Fork](#step-1-fork)
+    - [Step 2: Branch](#step-2-branch)
+    - [Step 3: Commit](#step-3-commit)
+    - [Step 5: Push](#step-5-push)
+    - [Step 6: Pull Request](#step-6-pull-request)
   - [Coding guidelines](#coding-guidelines)
-      - [Code structuring patterns](#code-structuring-patterns)
-          - [Object Oriented Programming](#object-oriented-programming)
-          - [Functional Programming](#functional-programming)
-      - [Naming patterns](#naming-patterns)
+    - [Code structuring patterns](#code-structuring-patterns)
+    - [Functional Programming](#functional-programming)
+    - [Naming conventions](#naming-conventions)
       - [Documentation](#documentation)
-      - [Writing tests](#writing-tests)
+    - [Writing tests](#writing-tests)
+
+## What and when to contribute
+
+You can contribute anything to the IF, but we are likely to close out unsolicited PRs without merging them. Our issue board is completely open and we have tags (`help-wanted`, `good-first-issue`) to help contributors to choose tasks to work on. We recommend speaking to the core team on Github before starting working on an issue. You can do this by raising an issue or commenting on an existing issue. This helps us to direct your energy is directions that are aligned with our roadmap, prevent multiple people working on the same task, and better manage our board. This all makes it much more likely that your work will get merged.
```

Comment:
@jmcook1186  Seems a small typo here
This helps us to direct your energy is directions -> This helps us to direct your energy in directions

Reply:
```suggestion
You can contribute anything to the IF, but we are likely to close out unsolicited PRs without merging them. Our issue board is completely open and we have tags (`help-wanted`, `good-first-issue`) to help contributors to choose tasks to work on. We recommend speaking to the core team on Github before starting working on an issue. You can do this by raising an issue or commenting on an existing issue. This helps us to direct your energy in directions that are aligned with our roadmap, prevent multiple people working on the same task, and better manage our board. This all makes it much more likely that your work will get merged.
```